pkg/autodiscover/autodiscover.go

```diff
@@ -111,6 +116,11 @@ func DoAutoDiscover() DiscoveredTestData {
 		logrus.Fatalf("Cannot load configuration, error: %v", err)
 	}
 	oc := clientsholder.GetClientsHolder()
+	data.AllNamespaces, _ = getAllNamespaces(oc.K8sClient.CoreV1())
+	data.AllSubscriptions = findSubscriptions(oc.OlmClient, []string{""})
+	data.AllCsvs = getAllOperators(oc.OlmClient)
+	data.AllInstallPlans, _ = getAllInstallPlans(oc.OlmClient)
+	data.AllCatalogSources, _ = getAllCatalogSources(oc.OlmClient)
```

Comment:
You're ignoring some errors here. Does it make sense calling the `createOperators()` func later when you have had errors when retrieving this related resources?

Reply:
fixed